numpy/lib/mixins.py

```diff
@@ -5,53 +5,54 @@
 
 from numpy.core import umath as um
 
-# None of this module should be exposed in top-level NumPy module.
+# Nothing should be exposed in the top-level NumPy module.
 __all__ = []
 
 
+def _disables_array_ufunc(obj):
```

Comment:
@shoyer I tend to agree with your position, OTOH, putting it in the class is easy to do and puts everything in one bundle. The fact that it is private should serve as warning to users that the warranty is void if they choose to fool with it.